Source/WebCore/dom/ExtensionStyleSheets.cpp

```diff
@@ -123,12 +123,12 @@ void ExtensionStyleSheets::updateInjectedStyleSheetCache() const
     m_injectedAuthorStyleSheets.clear();
     m_injectedStyleSheetToSource.clear();
 
-    Page* owningPage = m_document.page();
+    Page* owningPage = m_document->page();
     if (!owningPage)
         return;
 
     auto addStyleSheet = [&](const UserStyleSheet& userStyleSheet) {
-        auto sheet = createExtensionsStyleSheet(const_cast<Document&>(m_document), userStyleSheet.url(), userStyleSheet.source(), userStyleSheet.level());
+        auto sheet = createExtensionsStyleSheet(const_cast<Document&>(m_document.get()), userStyleSheet.url(), userStyleSheet.source(), userStyleSheet.level());
```

Comment:
Could we use `Ref` instead of `auto` for clarity?

Reply:
Sure.